openwisp_utils/admin_theme/templates/admin/change_form.html

```diff
@@ -0,0 +1,24 @@
+{% extends "admin/change_form.html" %}
+{% block admin_change_form_document_ready %}
+{{ block.super }}
+<script>
+(function ($) {
+    $(document).ready(function () {
+        var copyableFields = {{ copyable_fields | safe }} || [];
+        
+        copyableFields.forEach(copyableField => {
+            
+            var copyableFieldContainer = $(`.field-${copyableField} .readonly`).eq(0);
+            
+            copyableFieldContainer.html(`<input readonly id="id_${copyableField}" type="text" 
+            class="vTextField readonly" value="${copyableFieldContainer.text()}">`);
+            
+            var copyableFieldSelectedId = $(`#id_${copyableField}`);
+            copyableFieldSelectedId.click(function () {
+                $(this).select();
+            });
+        })
+    })
+}) (django.jQuery);
+</script>    
+{% endblock %}
```

Comment:
what about keeping here only the list of fields and moving the JS logic to a static file instead (which is going to be the same in the entire app and hence can be cached)?

Reply:
sure